exampleSite/content/index/copyright.md

```diff
@@ -0,0 +1,21 @@
++++
+fragment = "copyright"
+#disabled = true
+date = "2016-09-07"
+lastmod = "2017-09-07"
+weight = 1250
+#background = ""
+
+copyright = "" # default: Copyright $Year $params.name
+attribution = true # enable attribution by setting it to true
+
+[[menus]]
```

Comment:
I like that it moved into the fragment. Not sure, if that's the best thing to do in the long run. We are working around the integrated menu support in Hugo. That might be something to bite us once we have multi lingual sites etc. It would also make it harder to use the extended features set for menus. https://gohugo.io/content-management/menus/

Might be ok in the copyright footer, but for the navigation bar that might be different.
But even with it being ok in the footer, we would increase the inconsistencies within the various menus.

Thoughts?

Reply:
Fixed in 5cb5f94